### DISCUSSION TITLE
Acme.sh Error

### DISCUSSION THREAD

**Discussion by Crip-Zone** [acmesh-official/acme.sh#5079] [Q&A]
Hello, I am getting the following error in an email from my server. I'm certain it is because I've recently changed my domain from .com 'host-for.com' to .org 'host-for.org'.

> [Wed Mar 20 00:29:08 GMT 2024] Invalid status, piglet.host-for.com:Verify error detail:35.186.223.180: Invalid response from http://piglet.host-for.com/.well-known/acme-challenge/sZq4sapNrSSGWHTBbRUQwkQyDrQNj_Fpm2FGM0F8JZY: 500
[Wed Mar 20 00:29:08 GMT 2024] Please add '--debug' or '--log' to check more details.
[Wed Mar 20 00:29:08 GMT 2024] See: https://github.com/acmesh-official/acme.sh/wiki/How-to-debug-acme.sh
[Wed Mar 20 00:29:09 GMT 2024] Error renew piglet.host-for.com_ecc. 

Is there any way to change a domain in acme.sh? Or do I need to uninstall it and reinstall it with the new domain? Any help would be appreciated.

Thanks,
Steve

**Comment by neilpang** [comment]
use `--issue` to issue a new one.
